.release-please-manifest.json

```diff
@@ -7,5 +7,6 @@
   "libs/providers/config-cat": "0.4.0",
   "libs/providers/launchdarkly-client": "0.2.0",
   "libs/providers/go-feature-flag-web": "0.1.5",
-  "libs/shared/flagd-core": "0.1.8"
+  "libs/shared/flagd-core": "0.1.8",
+  "libs/providers/flipt": "0.1.0"
```

Comment:
does this need to match the version in the `package.json`?

Reply:
@markphelps Good call! thank you.